app/main.py

```diff
@@ -1,3 +1,72 @@
-def shop_trip():
-    # write your code here
-    pass
+from datetime import datetime
+from app.car import PriсeKm
+from app.customer import Customer
+from app.shop import Shop
+
+
+def shop_trip() -> str:
```

Comment:
try to separate this code to a few functions with different responsibilities, it is hard to read